packages/infra/lib/api-stack.ts

```diff
@@ -1256,6 +1266,39 @@ export class APIStack extends Stack {
     return fhirToMedicalRecordLambda;
   }
 
+  private setupFhirBundleLambda(ownProps: {
+    lambdaLayers: LambdaLayers;
+    vpc: ec2.IVpc;
+    fhirServerUrl: string;
+    envType: EnvType;
+    sentryDsn: string | undefined;
+    alarmAction: SnsAction | undefined;
+  }): Lambda {
+    const { lambdaLayers, vpc, fhirServerUrl, sentryDsn, envType, alarmAction } = ownProps;
+
+    const lambdaTimeout = MAXIMUM_LAMBDA_TIMEOUT.minus(Duration.seconds(5));
+
+    const fhirToBundleLambda = createLambda({
+      stack: this,
+      name: "FhirToBundle",
+      runtime: lambda.Runtime.NODEJS_18_X,
+      entry: "fhir-to-bundle",
+      envType,
+      envVars: {
+        FHIR_SERVER_URL: fhirServerUrl,
+        ...(sentryDsn ? { SENTRY_DSN: sentryDsn } : {}),
+      },
+      layers: [lambdaLayers.shared, lambdaLayers.chromium],
```

Comment:
Do we need chromium there? I thought it was only retrieving resources from the FHIR server, no?